scss/components/_vr-controls.scss

```diff
@@ -1,21 +1,21 @@
 .oo-vr-icon {
   &--move {
     position: absolute;
-    height: 15px;
-    width: 15px;
+    height: calc(#{$vr-controls-full-size}/4);
```

Comment:
Shouldn't we just adjust the size of the icon instead of this resizing?

Reply:
Am I undestanding this correctly? When the camera is stationary, the icons for vr are bigger. When you are moving they shrink.  Is that correct?

Also, can you divide by a constant instead of 4. That way we can adjust it easily if we decide to change the size.  You can do it in scss/base/_variables.scss where you have vr-controls-full-size defined.

Reply:
fix